polaris.shopify.com/src/components/Layout/Layout.module.scss

```diff
@@ -17,6 +17,10 @@
   @media screen and (max-width: $breakpointTablet) {
     display: block;
   }
+
+  @media screen and (max-width: $breakpointMobile) {
+    padding: 0;
```

Comment:
This fixes small screen being doubled up, as `Container` also adds `1rem` of padding.